src/tests/java/picard/util/IlluminaUtilTest.java

```diff
@@ -73,15 +73,4 @@
             }
         };
     }
-
-    /*
-    @Test(dataProvider = "solexaQualStrToPhreds")
```

Comment:
Ditto here - not sure why this is commented out.  Should probably be uncommented and disabled instead of removing it.